teachopencadd/README.md

```diff
@@ -3,19 +3,19 @@
 This index gives an overview on the available TeachOpenCADD talktorials. 
 
 - `000_template`  # new
-- `001_chembl_compound_query`  # T1
-- `002_compound_filtering_adme`  # T2
-- `003_compound_filtering_unwanted_substructures`  # T3
-- `004_ligand_based_screening_compound_similarity`  # T4
+- `001_chembl_query`  # T1
+- `002_compound_adme`  # T2
+- `003_compound_unwanted_substructures`  # T3
+- `004_compound_similarity`  # T4
 - `005_compound_clustering`  # T5
 - `006_compound_maximum_common_substructures`  # T6
-- `007_ligand_based_screening_machine_learning`  # T7
-- `008_pdb_protein_query`  # T8
+- `007_ligand_based_machine_learning`  # T7
+- `008_pdb_query`  # T8
 - `009_ligand_based_pharmacophores`  # T9
 - `010_binding_site_comparison`  # T10
 - `011_online_api_introduction`  # T11
```

Comment:
In hindsight, not all of those are APIs strictly speaking, so maybe we can instead use: 

```suggestion
- `011_online_api_webservices`  # T11
```